src/patternfly/components/Label/label.scss

```diff
@@ -277,6 +313,13 @@
     }
   }
 
+  .pf-c-label__editable-text {
+    @include pf-text-overflow;
+
+    max-width: var(--pf-c-label__editable-text--MaxWidth);
+    border-width: var(--pf-c-label__editable-text--BorderWidth);
```

Comment:
this is fine but I think it's also fine just to use `border: 0;` since I wouldn't expect it to be a themeable thing.

Reply:
I wondered if someday the active state would want a border other than the focus ring. 🤷